samples/bootloader/src/main.c

```diff
@@ -122,7 +123,11 @@ static void boot_from(const struct fw_info *fw_info)
 	__set_MSP(vector_table[0]);
```

Comment:
vector_table is stored in stack, and you may have changed the stack pointer before accessing this.
Didn't look at it super carefully, but it seems dangerous to me...

Reply:
Not part of this PR I think maybe it should be a bit improved now where it's moved down after but I haven't looked into it.